### ISSUE TITLE
No valid domains remaining.

### ISSUE THREAD

**Issue by b11r1m** [d3mondev/puredns#57]
Resolving domains with public resolvers
Processed: 79 Rate: 79 Elapsed: 00:00:25

Detecting wildcard root subdomains
[ETA 00:00:00] |░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░░| 0/0 queries: 0 (time: 00:00:00)

Validating domains against trusted resolvers
Processed: 0 Rate: 0 Elapsed: 00:00:00

No valid domains remaining.

-----------------------------
               puredns v2.1.1

**Comment by Thamaraiselvan942** [comment]
+1

**Comment by cyberduck404** [comment]
same

**Comment by asxjoy** [comment]
agreed, dnsx shows lots of valid domains while puredns provides nothing with live custom resolvers
